azurerm/internal/services/eventgrid/tests/eventgrid_event_subscription_resource_test.go

```diff
@@ -355,6 +418,116 @@ resource "azurerm_eventgrid_event_subscription" "test" {
 `, data.RandomInteger, data.Locations.Primary, data.RandomInteger, data.RandomInteger, data.RandomInteger)
 }
 
+func testAccAzureRMEventGridEventSubscription_serviceBusQueue(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+resource "azurerm_servicebus_namespace" "example" {
```

Comment:
and put new lines here to be consistent with the restof the tests
```suggestion
}

resource "azurerm_servicebus_namespace" "example" {
```